assignment06/index.php

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title> Assignment 6 </title>
+    <meta charset="utf-8">
+    <!--
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    <script src="js/myScript.js"></script>
+    -->
+</head>
+
+<body>
+    <!--
+        James Welu
+        assignment02.html
+        INFO2340.WW
+        Thoendel
+        01/20/2020
+    -->
+
+<h1>Hello, world!</h1>
+<div>Allow me to introduce myself! My name is <?php echo "James Welu"; ?>!</div>

```

Comment:
solid php statements! complete with quotation marks and those pesky semicolons! 👍